NETWORK/NetworkAddPedToSynchronisedScene.md

```diff
@@ -11,26 +11,27 @@ void NETWORK_ADD_PED_TO_SYNCHRONISED_SCENE(Ped ped, int netScene, char* animDict
 Adds a ped to a networked synchronised scene.
 
 Synchronized scene playback flags (Also works in other `NETWORK_ADD_*_TO_SYNCHRONISED_SCENE` natives):
-|Value| Name | Notes |
-|:----:|:------:| :--------: |
-|`0`| None | No flag set. |
-|`1`| USE_PHYSICS | Allows the ped to have physics during the scene. |
-|`2`| TAG_SYNC_OUT | The task will do a tag synchronized blend out with the movement behaviour of the ped. |
-|`4`| DONT_INTERRUPT | The scene will not be interrupted by external events. |
-|`8`| ON_ABORT_STOP_SCENE | The scene will be stopped if the scripted task is aborted. |
-|`16`| ABORT_ON_WEAPON_DAMAGE | The scene will be stopped if the ped is damaged by a weapon. |
-|`32`| BLOCK_MOVER_UPDATE | The task will not update the mover. |
-|`64`| LOOP_WITHIN_SCENE | Animations within this scene will be looped until the scene is finished. |
-|`128`| PRESERVE_VELOCITY | The task will keep it's velocity when the scene is cleaned up/stopped. Do note that the `USE_PHYSICS` flag must also be present. |
-|`256`| EXPAND_PED_CAPSULE_FROM_SKELETON | The task will apply the `ExpandPedCapsuleFromSkeleton` reset flag to the ped (See [`SET_PED_RESET_FLAG`](#_0xC1E8A365BF3B29F2)). |
-|`512`| ACTIVATE_RAGDOLL_ON_COLLISION | The ped will be ragdoll if it comes in contact with an object. |
-|`1024`| HIDE_WEAPON | The ped's current weapon will be hidden during the scene. |
-|`2048`| ABORT_ON_DEATH | The synchronised scene will be aborted if the ped dies. |
-|`4096`| VEHICLE_ABORT_ON_LARGE_IMPACT | If the scene is running on a vehicle, then it will be aborted if the vehicle takes a heavy collision with another vehicle. |
-|`16384`| PROCESS_ATTACHMENTS_ON_START | Attachments will be processed at the start of the scene. |
-|`32768`| NET_ON_EARLY_NON_PED_STOP_RETURN_TO_START |  A non-ped entity will be returned to their starting position if the scene finishes early. |
-|`65536`| SET_PED_OUT_OF_VEHICLE_AT_START | If the ped is in a vehicle when the scene starts, it will be set out of the vehicle. |
-|`131072`| NET_DISREGARD_ATTACHMENT_CHECKS | Attachment checks will be disregarded when the scene is running. |
+| Value     |                  Name                     |                                                             Notes                                                                |
+| :-------: | :---------------------------------------: | :------------------------------------------------------------------------------------------------------------------------------: |
```

Comment:
Are you sure that the :- and -: will parse properly? I've not seen them used in markdown before.

Reply:
I was following the example from ENABLE_MOON_CYCLE_OVERRIDE hmm

Reply:
In standard MD, you can align columns in a table using colons (`:`) inside the pipes (`|`).  You can check [OverrideVehHorn](https://github.com/citizenfx/natives/edit/master/AUDIO/OverrideVehHorn.md) to see it's [supported](https://docs.fivem.net/natives/?_0x3CDC1E622CCE0356).

Reply:
so is everything okay? because im so confused why it doesnt parse, only thing wrong was the missing flag if im not mistaken

Reply:
It's because you did `|Value|``instead of `| Value |` so it didn't parse correctly. :)

Reply:
> It's because you did `|Value|` instead of `| Value |` so it didn't parse correctly. :)
> 
> In Markdown, the standard format for a table header is with spaces surrounding the text to separate the pipes

That's what i was suspcious about too! Thank you so much